views.py

```diff
@@ -83,6 +83,7 @@ def indexView(page: ft.Page, params: Params, basket: Basket):
 def museumInfoView(page: ft.Page, params: Params, basket: Basket):
     id = int(params.get('id'))
     museum = Museum.get_by_id(id)
+    mapcoordin = MapCoordinates.get_by_id(id)
```

Comment:
Разве мы не может получить координаты у музея через связнное поле?
```suggestion
coordinates = museum.coordinates
```

Reply:
а оно так робить будет?